mobile/library/objective-c/EnvoyHTTPStreamImpl.mm

```diff
@@ -130,7 +130,7 @@ static void ios_on_error(envoy_error error, envoy_stream_intel stream_intel,
 @implementation EnvoyHTTPStreamImpl {
   EnvoyHTTPStreamImpl *_strongSelf;
   EnvoyHTTPCallbacks *_platformCallbacks;
-  envoy_http_callbacks _nativeCallbacks;
+  ios_context *_context;
```

Comment:
i'm not clear on `ios_context` and why it's needed... I thought `envoy_http_callbacks` would be replaced by `EnvoyStreamCallbacks`?

Reply:
The `ios_context` is needed for the actual callback implementation: https://github.com/envoyproxy/envoy/blob/5cf7e081adecab59fa245ca889ddba108b0a6f58/mobile/library/objective-c/EnvoyHTTPStreamImpl.mm#L29

The existing code is a bit strange that has multiple callback implementations. This is the same issue with the Android code as well. We should clean it up at some point.